data/fwupd.conf

```diff
@@ -0,0 +1,5 @@
+# use `man 5 fwupd.conf` for documentation
```

Comment:
can we also put the markdown into fwupd.github.io and leave a URL here for the manpage hating people like me?

Reply:
Done.